quotient/quotient/main.c

```diff
@@ -0,0 +1,36 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+
+int main() {
+    int firstNumber = 0;
+    int secondNumber = 0;
+    int answerSign = 1;
+    printf("Enter the first number: ");
+    scanf_s("%d", &firstNumber);
+    printf("\nEnter the second number: ");
+    scanf_s("%d", &secondNumber);
+    int absQuotient = 0;
+    while (secondNumber == 0) {
+        printf("You can't divide by zero :c\nTry again\n");
+        printf("Enter the second number : ");
+        scanf_s("%d", &secondNumber);
+    }
+
+    if (abs(firstNumber) < abs(secondNumber)) {
+        printf("The answer is: %d", absQuotient);
```

Comment:
```suggestion
        printf("The answer is: %d", 0);
```
Так читать и понимать проще